components/Carousel/Carousel.js

```diff
@@ -1,9 +1,54 @@
 class Carousel {
+    constructor(effect){
```

Comment:
good start to the sprint, maybe come back and finish it since you have the knowledge and know how now